### PR TITLE
SMV version should be hardcoded in smv-install (#818)

### PR DESCRIPTION
**Pull request by laneb**
Addressing #818.

### CODE REVIEW

File: tools/smv-install [hunk 1]
```diff
@@ -21,7 +19,7 @@ function parse_args()
     DO_INSTALL_SPARK=1
     shift
   fi
-  SMV_VERSION="$1"
+  SMV_VERSION="1.5.2.9"
```
Comment (jacobdr) [inline tools/smv-install]: I think that this should be user-configurable if desired. So something like:

`SMV_VERSION="${SMV_VERSION:-1.5.2.9}"`

Previously, I have used the smv-install script from the prior release to download a new version at a different install location on disk, prepended it to my PATH, and then tested the new version against our test suite. This PR in its current form makes it harder to do that I think

Comment (laneb) [inline tools/smv-install]: It is true that you would not be able to use e.g. the 1.5.2.10 script to install 1.5.2.11. Can you elaborate on why that would be preferable over following the installation instructions for 1.5.2.11?

Comment (AliTajeldin) [inline tools/smv-install]: There is more to the hardcoding of the version in the install script than just not requiring the user to supply the version on the command line.  We shouldn't have to make the script backwards compatible with old versions.
To install older versions, we can just substitute the desired version number in the `curl` command used to download and run the script in the README.  We should document how to install older versions.

Comment (AliTajeldin) [inline tools/smv-install]: put comment above this to indicate that it is automatically set by the `smv-release` script.

**Comment by AliTajeldin** [comment]
Merging right now to make smv release.  Add comment to master later.

File: tools/smv-install (hunk 1 above)
Comment (laneb) [inline tools/smv-install]: Done.

**Pull request merged**